src/pages/help-centre/components/_didnt-find-answer.tsx

```diff
@@ -84,15 +62,6 @@ const DidntFindYourAnswerBanner = () => {
                         <Button secondary onClick={() => LC_API.open_chat_window()}>
                             <Localize translate_text="Chat" />
                         </Button>
-                        <WhatsAppButton onClick={() => window.open(whatsapp_url, '_blank')}>
```

Comment:
@niloo-fs Not sure why you have deleted the WhatsApp button here as we need to have it 🤔 

Anyway, To fix the issue we need to remove the hardcode width at line [24](https://github.com/binary-com/deriv-com/pull/3921/files#diff-97605a2cd0158d762678dcd27d355b0be864ae3afc1f8d87f6d567fd5e2ad415R24) and replace it with `flex: 1` and also give the parent element `flex: 1`.

Reply:
@mitra-deriv as I checked in the production we don't have whatsapp icon there, 
they removed that and used the icon under livechat button
because of that I removed the button

Reply:
@niloo-fs We actually have it in production but showing to certain countries, also we are now showing to all countries the card for this changes is in current tag and you can check it on staging thanks 🙏

Reply:
Oh got it, I thought we removed that. will change this